test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -1400,3 +1400,17 @@
     (testing "and the file doesn't exist"
       (testing "the result is nil"
         (is (nil? (create-csr-attrs-exts "does/not/exist.yaml")))))))
+
+(deftest ca-expiration-dates-test
+  (testing "returns a map of names to dates"
+    (let [settings (testutils/ca-sandbox! bundle-cadir)
+          expiration-map (ca-expiration-dates (:cacert settings))]
+      (is (= "2036-09-06T05:58:33UTC" (get expiration-map "rootca.example.org")))
+      (is (= "2036-09-06T06:09:14UTC" (get expiration-map "intermediateca.example.org"))))))
+
+(deftest crl-expiration-dates-test
+  (testing "returns a map of names to dates"
+    (let [settings (testutils/ca-sandbox! bundle-cadir)
+          expiration-map (crl-expiration-dates (:cacrl settings))]
+      (is (= "2016-10-11T06:42:52UTC" (get expiration-map "rootca.example.org")))
```

Comment:
Is the UTC timestamp suffix the norm for puppetserver?  Most of the PE apps use "Z" for the suffix.

Reply:
I used the existing `time-format` methods in this codebase, so yes, I think it is. But I can return something different if you'd prefer. If so, an example formatter from one of the apps you're thinking of would be very helpful.

Reply:
I think we try to align with ISO-8601 as much as possible.  https://en.wikipedia.org/wiki/ISO_8601#Time_zone_designators

Reply:
I'm trying to find an example in one of the other repos.

Reply:
Specifically, we do it this way for compatability with the Ruby, probably a holdover from the old CA. See https://github.com/puppetlabs/puppetserver/blob/master/src/clj/puppetlabs/puppetserver/certificate_authority.clj#L557. But I don't have an issue with returning that format from this endpoint.

Reply:
Likely best to stick with tradition.

Reply:
If that's the format used in other puppetserver APIs, I think sticking with it makes sense.

Reply:
Looks like it is what's used to return `not-after` dates [for agent certs](https://github.com/puppetlabs/puppetserver/blob/master/src/clj/puppetlabs/puppetserver/certificate_authority.clj#L1424-L1434) via the API.